examples/cloudfront-lambda-urls/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "cloudfront-lambda-urls",
+    "devDependencies": {
+        "@types/node": "^10.0.0"
+    },
+    "dependencies": {
+        "@pulumi/aws": "^4.6.0",
+        "@pulumi/aws-native": "^0.117.0",
```

Comment:
Should we use the current major now? I guess there shouldn't be anything stopping us from doing that, right?

Reply:
Good catch.